modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-10-0_{context}"]
+= Release notes for {gitops-title} 1.10.0
+
+{gitops-title} 1.10.0 is now available on {product-title} 4.12, 4.13 and 4.14.
```

Comment:
Do we want to get 4.14 involved as its not GA yet

Reply:
Well I was told to by @reginapizza to do so.

Reply:
@masauso-lungu 4.14 is not yet GA. we will have to have a 1 pointer JIRA issue to change it here later after 4.14 is out and LIVE.

Please see this comment: https://github.com/openshift/openshift-docs/pull/65102#discussion_r1336782442

Reply:
Noted!, i did not see that comment. Removed now.